api/envoy/service/status/v3/csds.proto

```diff
@@ -88,7 +90,13 @@ message PerXdsConfig {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.service.status.v2.PerXdsConfig";
 
+  // Config status is generated by management servers.
```

Comment:
Please add "Will not be present if the CSDS server is an xDS client."

Reply:
Comment added.